modules/git/repo_commitgraph_gogit.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	gitealog "code.gitea.io/gitea/modules/log"
 
-	"github.com/go-git/go-git/v5/plumbing/format/commitgraph"
+	commitgraph_v2 "github.com/go-git/go-git/v5/plumbing/format/commitgraph/v2"
```

Comment:
Wow. I never thought I'd see two version numbers in the same import anytime.

Reply:
I see some failed tests with newer gogit, so I think it's better to revert it to the old version, then leave enough time to figure out how to upgrade gogit in the future.